test/builder/SubscriptionsBuilder.scala

```diff
@@ -36,7 +37,7 @@ trait SubscriptionsBuilder {
     }
 
   def buildSubscriptionFieldsWrapper(applicationId: ApplicationId, fields: Seq[SubscriptionFieldValue] = Seq.empty) = {
-    SubscriptionFieldsWrapper(applicationId, s"clientId-$applicationId", s"context-$applicationId", s"apiVersion-$applicationId", fields = fields)
+    SubscriptionFieldsWrapper(applicationId, ClientId(s"clientId-$applicationId"), s"context-$applicationId", s"apiVersion-$applicationId", fields = fields)
```

Comment:
probably want .value here - or the text will be overly verbose.  Not that it would break anything

Reply:
Could you explain a little more - we are struggling to understand what you mean :(

Reply:
ClientId(s"clientId-${applicationId.value}) and suchlike.  not part of the work you just did but the bit before.  only now spotted it because the clientId construct highlights it.

Reply:
Oh you mean the applicationId?